### DISCUSSION TITLE
Cylinder Lock is not detected by HA

### DISCUSSION THREAD

**Discussion by razvanelgsm** [jbergler/hass-ttlock#142] [Q&A]
The device was bought from here:

https://www.aliexpress.com/item/1005004569536382.html

and the lock number is BMC_e07e5c

In TTlock app is working without issues just to make it working with the gateways (G2 and G3)
was need to update them to the latest firmware.

**Comment by mremedi2023** [comment]
Hi, after doing many tests I have lost my 2 keyboards that worked without problems. I tried creating a new account on the ttlock website, linked the 2 keyboards to that account but the integration does not see them either, what could be happening? Thanks

**Comment by mremedi2023** [comment]
Sorry, but I hadn't realized that the G3 gateway had disappeared from the app. By the way, there's no way to pair it again, it never finishes adding it, it gives an error, and I don't see a reset button on it.

**Comment by jbergler** [comment]
I'm sorry, but I have no idea how to help you.
I would suggest getting everything setup properly in the ttlock app on your phone, and then when it's working properly there try setting the home assistant integration up again.
